### PR TITLE
Feat: create the Footer component by Ibrahim

### LINKED ISSUE #8: Component > Footer
Design the footer component
inside components create a new folder `Footer` and then `Footer.js`

image

### PR DESCRIPTION
**Pull request by Nourcova**
This is the Footer component, It looks good for me, if you have anything please let me know :)

### CODE REVIEW

File: src/Components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import logoFooter from '../../assets/images/logoFooter.png';
+import dribbble from '../../assets/images/Dribbble.png';
+import instagram from '../../assets/images/Instagram.png';
+import twitter from '../../assets/images/Twitter.png';
+import youtube from '../../assets/images/Youtube.png';
+
+function Header() {
+  return (
+    <div
+      style={{ backgroundColor: '#4FAA4D' }}
```
Comment (JanaKhanji) [inline src/Components/Footer/Footer.js]: I guess u can add a class "bg-primary" instead of applying a style

Comment (Nourcova) [reply src/Components/Footer/Footer.js]: Noted, I changed it and pushed it again 
And I added the © to the footer too

File: src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import logoFooter from '../../assets/images/logoFooter.png';
+import dribbble from '../../assets/images/Dribbble.png';
+import instagram from '../../assets/images/Instagram.png';
+import twitter from '../../assets/images/Twitter.png';
+import youtube from '../../assets/images/Youtube.png';
+
+function Header() {
```
Comment (osamaakb) [inline src/Components/Footer/Footer.js]: name is Footer
```suggestion
function Footer() {
```

File: src/Components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import logoFooter from '../../assets/images/logoFooter.png';
+import dribbble from '../../assets/images/Dribbble.png';
```
Comment (osamaakb) [inline src/Components/Footer/Footer.js:3]: these aren't in assets? can you push them too

File: src/Components/Footer/Footer.js (hunk 1 above)
Comment (osamaakb) [inline src/Components/Footer/Footer.js]: try to avoid inline-styles, you can achieve this style by adding `bg-primary` to the classNames

**Review by osamaakb — CHANGES_REQUESTED** [review]

**Review by osamaakb — APPROVED** [review]

**Pull request merged**